### DISCUSSION TITLE
定时发送语音消息

### DISCUSSION THREAD

**Discussion by A2Sumie** [Tsuk1ko/CQ-picfinder-robot#457] [一般讨论]
我手上有一些mp3希望按照定时消息的方式作为早安发出去
一下子没有想到只用cqps怎么设置（现在是用cqps拉dice!这种扭曲方法）

（以及有没有可能设置，指定时间后，抽选一种reply进行回复？）

**Comment by A2Sumie** [comment]
看了下语言库功能里有一个`<replace>`，这个能给定时功能用吗

**Reply by Tsuk1ko** [reply]
不能

**Comment by Miuzarte** [comment]
[https://github.com/Tsuk1ko/cq-picsearcher-bot/wiki/附加功能#防止 CQ 转义](https://github.com/Tsuk1ko/cq-picsearcher-bot/wiki/%E9%99%84%E5%8A%A0%E5%8A%9F%E8%83%BD#%E9%98%B2%E6%AD%A2-cq-%E8%BD%AC%E4%B9%89)

最好提前转成amr，没法保证qq端会自动转码

**Comment by Tsuk1ko** [comment]
感觉你这个比较个人化的需求的话，不如直接写一个脚本网络请求机器人客户端的 http api，然后服务器 cron 去触发
